libs/elasticsearch/tests/_sync/fake_embeddings.py

```diff
@@ -13,7 +14,9 @@ class FakeEmbeddings(Embeddings):
     def embed_documents(self, texts: List[str]) -> List[List[float]]:
         """Return simple embeddings.
         Embeddings encode each text as its index."""
-        return [[float(1.0)] * 9 + [float(i)] for i in range(len(texts))]
+        temp = [[float(1.0)] * 9 + [float(i)] for i in range(len(texts))]
+        print(temp)
```

Comment:
You left a print statement here. Also note that the files in the `_sync` directory are not written by hand, we have a process that generates them from the versions in _async. See the `make run_unasync` command.